roles/letsencrypt/README.md

```diff
@@ -102,7 +101,7 @@ wildcard-certificates use base name again as otherwise DNS txt record creation c
 | private_key_path                         | yes      | $letsencrypt_conf_dir/certs          | path for private key
 | remaining_days                           | yes      | 30                                   | min days remaining before certificate will be renewed
 | convert_cert_to                          | no       |                                      | format to convert the certificate to: `pfx`
-
+| acme_directory_validate_certs            | no       | true                                 | Only exists to enable integration tests of http challenges with pebble server
```

Comment:
We already have a variable for this here:
https://github.com/T-Systems-MMS/ansible-collection-letsencrypt/blob/master/roles/letsencrypt/tasks/dns-challenge-pebble.yml#L40
https://github.com/T-Systems-MMS/ansible-collection-letsencrypt/blob/master/tests/integration/targets/letsencrypt/test.yml#L20

Reply:
fixed with 41f3150